lib/js/components/CounterToggle/CounterToggle.vue

```diff
@@ -257,14 +262,19 @@ $counter-toggle-colors: (
 	}
 
 	&__icon {
-		margin-right: $space-xxxxxs;
 		transition: color ease-in-out $default-transition-time;
 	}
 
 	&__counter {
 		@include textBold();
 		@include buttonS();
 	}
+
+	&:not(.-single-item) {
```

Comment:
`-iconOnly`?

Reply:
Actually, is this class even needed? We don't render `.counterToggle__counter` when there is no counter so there won't be no margin applied anyway.